connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java

```diff
@@ -244,6 +244,14 @@
             + "user requests to reset the set of active topics per connector.";
     protected static final boolean TOPIC_TRACKING_ALLOW_RESET_DEFAULT = true;
 
+    /**
+     * @link "https://www.eclipse.org/jetty/documentation/current/header-filter.html"
+     * @link "https://www.eclipse.org/jetty/javadoc/9.4.28.v20200408/org/eclipse/jetty/servlets/HeaderFilter.html"
+     **/
+    public static final String RESPONSE_HTTP_HEADERS_CONFIG = "response.http.headers.config";
+    public static final String RESPONSE_HTTP_HEADERS_DOC = "Set values for Jetty HTTP response headers";
```

Comment:
I don't think we should expose `Jetty` here. Yes, we're following the Jetty grammar and format for these, but let's not unnecessarily expose the internals.
```suggestion
    public static final String RESPONSE_HTTP_HEADERS_DOC = "Rules for REST API HTTP response headers";
```